flask/main.py

```diff
@@ -73,19 +75,19 @@ def success():
 
 @app.route('/products', methods=['GET'])
 def products():    
-    print('/products')
     try:
-        rows = get_products()
+        with sentry_sdk.start_span(op="/products.get_products", description="function"):
```

Comment:
putting `/products....`so it's clear that this is the endpoint name and that get_products is inside of it.

if this seems redundant or too verbose, I'm happy to remove it.

possibilities:
```
with sentry_sdk.start_span(op="/products.get_products", description="function"):
with sentry_sdk.start_span(op="products.get_products", description="function"):
with sentry_sdk.start_span(op="get_products", description="function"):
```
Titling your spans effectively can reveal the structure of your code that was invoked (without having to show the source code via something like a stacktrace).

![image](https://user-images.githubusercontent.com/8920574/126825114-eafc3097-1fa4-41f6-b8fc-4f2f7e7a7b99.png)